source/models/database.py

```diff
@@ -0,0 +1,55 @@
+import os, json
+
+from commands_functions.schema_keys import Keys
+from models.table import Table
+from outputs.exceptions import *
+
+class Database:
+    def __init__(self, DB_name = None, schema_path = None):
+        if schema_path is not None:
+            self.schema_data = self.validate_schema_path(schema_path)
```

Comment:
Database doesn't have an attribute called schema_data. you create the list of tables by schema_data. you should't do `self.schema_data`

Reply:
i did so in the next lines but thought we don't need it here.
i solved it